must.go

```diff
@@ -0,0 +1,31 @@
+package container
+
+func MustSingleton(c Container, resolver interface{}) {
+	if err := c.Singleton(resolver); err != nil {
+		panic(err)
+	}
+}
+
+func MustNamedSingleton(c Container, name string, resolver interface{}) {
+	if err := c.NamedSingleton(name, resolver); err != nil {
+		panic(err)
+	}
+}
+
+func MustTransient(c Container, resolver interface{}) {
+	if err := c.Transient(resolver); err != nil {
+		panic(err)
+	}
+}
+
+func MustNamedTransient(c Container, name string, resolver interface{}) {
+	if err := c.NamedTransient(name, resolver); err != nil {
+		panic(err)
+	}
+}
+
+func MustCall(c Container, receiver interface{}) {
+	if err := c.Call(receiver); err != nil {
+		panic(err)
+	}
+}
```

Comment:
These methods make container configuration in main() much neater.

Here's some code from my current application:

```golang
func main() {
	c := container.New()

	container.MustSingleton(c, func() auth.RequestAuth {
		return auth.New("ids")
	})

	container.MustSingleton(c, func(db *ent.Client, auth auth.RequestAuth) protos.AccountsServer {
		return &accounts{
			db:   db,
			auth: auth,
		}
	})

	container.MustSingleton(c, func(db *ent.Client, auth auth.RequestAuth) protos.ApplicationsServer {
		return &applications{
			db:   db,
			auth: auth,
		}
	})

	container.MustSingleton(c, func(cfg *config.Config, db *ent.Client, applications protos.ApplicationsServer, accounts protos.AccountsServer) ApiServer {
		return &server{
			enableReflection: cfg.GrpcReflection,
			db:               db,
			applications:     applications,
			accounts:         accounts,
		}
	})

	container.MustCall(c, func(server ApiServer) error {
		return server.ListenAndServe("0.0.0.0:8000")
	})
}

```